src/components/miniplayer/View.tsx

```
@@ -0,0 +1,131 @@
+      return collapse();
+    }
+    if (translationY < -height * SnapToRatio) {
+      return expand();
+    }
+    return (miniplayerHeight.value = withTiming(initHeight.value, {
+      duration: 500,
+    }));
+  };
+
+  const scrollDragGesture = Gesture.Pan()
+    .onStart(e => (initHeight.value = miniplayerHeight.value))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)